src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest

```diff
@@ -10,8 +10,7 @@
     <Tags>code analysis; roslyn; analyzer; analyzers; live; fxcop; code quality;</Tags>
   </Metadata>
   <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,]" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.5.0,]" />
```

Comment:
I do not believe we change the visual studio version non-foundational updates.  So 15.3 or 15.6 are valid but not 15.5

Reply:
Actually looking at http://versionof.net/ it does not appear to change the minor version number ever. restricting to 15.4.0 would be `15.0.27004.2002` I do not know what the number for 15.5 will be yet.

Reply:
Ah, so in that case I will change it to 15.3

Reply:
> it does not appear to change the minor version number ever

Thanks, I will close the PR for now and create a separate one once we know the 15.5 version.